lib/GADS/Schema/Result/Report.pm

```diff
@@ -304,12 +323,31 @@ sub create_pdf
         fg_color  => '#ffffff',
     };
 
-    $pdf->table(
-        data         => $fields,
-        header_props => $hdr_props,
-        border_c     => '#007c88',
-        h_border_w   => 1,
-    );
+    foreach my $topic (keys %$grouped_data) {
+        next if $topic eq 'Other';
+        my $fields = [ [$topic, ''] ];
+        push( @{$fields}, [ $_->{name}, $_->{value} ] ) foreach (@{$grouped_data->{$topic}});
+
+        $pdf->table(
+            data         => $fields,
+            header_props => $hdr_props,
+            border_c     => '#007c88',
+            h_border_w   => 1,
+        );
+    }
+
+    foreach my $topic (keys %$grouped_data) {
+        next unless $topic eq 'Other';
+        my $fields = [ [$topic, ''] ];
+        push( @{$fields}, [ $_->{name}, $_->{value} ] ) foreach (@{$grouped_data->{$topic}});
+
+        $pdf->table(
+            data         => $fields,
+            header_props => $hdr_props,
+            border_c     => '#007c88',
+            h_border_w   => 1,
+        );
+    }
```

Comment:
Sorry if I'm being stupid, the it looks like the same tables are printed twice?

Reply:
This is to print only the fields within the 'Other' topic (see other comment). I have updated this to make it clearer.